homeassistant/components/tolo/select.py

```diff
@@ -14,36 +17,77 @@
 from .const import DOMAIN
 
 
+@dataclass(frozen=True, kw_only=True)
+class ToloSelectEntityDescription(SelectEntityDescription):
+    """Class describing TOLO select entities."""
+
+    options: list[str]
```

Comment:
Isn't this required by default as well?

Reply:
It is, but the existing definition allows options to be `None`. With this override, I require the TOLO selects to always provide `list[str]`. This helps with the `options` property, which requires a list but does not allow for `None`, so with this enforcement I don't need any checks for the `options` value from my custom entity description. Thought this would be the cleanest way. Is that ok?